docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/01 - LORIS Install/Troubleshooting.md

```diff
@@ -6,16 +6,17 @@ This guide offers support for common issues faced when first installing LORIS. I
 correctly followed and that all required dependencies have been installed.
 
 ## Frontend Does Not Load/JavaScript changes not visible
-The first thing to try is to run:
+The first command to try to run (under the _loris root_ directory):
 
 ```bash
-make
+make clean
 ```
 
-Alternatively, on development instances:
+If this does not solve the issue, you can try one of the following commands, depending on your environment:
```

Comment:
```suggestion
Next, run one of the following commands depending on your environment:
```

`make clean` is more of a preparatory step than a replacement for one of the following.